elements/pl-dataframe/pl-dataframe.py

```diff
@@ -163,7 +158,10 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     html_params = {
         "uuid": pl.get_uuid(),
         "frame_html": frame_style.to_html(),
-        "code_string": get_python_string(display_variable_name, frame),
+        "code_string": pprint.pformat(
+            frame.to_dict(), width=500, indent=4, sort_dicts=False
```

Comment:
Width seems a bit arbitrary. Did we want to set that as a possible option? 

The `indent` I think matches with the current linters?

Reply:
It's easy enough to set that I can add it as an option. 

Yeah, 4 space indent matches what Black uses.